config/default_config.yml

```diff
@@ -22,7 +22,7 @@ app:
 # Services
 services:
   geocoder:
-    url: override_by_environment
+    url: http://localhost/autocomplete
```

Comment:
This looks like something you changed on your local environment that shouldn't be merged to master :)